code/modules/food_and_drinks/food/foods/ingredients.dm

```diff
@@ -24,8 +24,8 @@
 
 /obj/item/reagent_containers/food/snacks/soydope
 	name = "soy dope"
-	desc = "Like regular dope, but for the health concious consumer."
 	icon_state = "soydope"
+	desc = "Like regular dope, but for the health concious consumer."
```

Comment:
Just a nitpick but I think it was better before, with the description below the name like most other foods

Reply:
Just tried to get it the same with the related group
Across all files its inconsistent, this is just the least locally inconsistent way of doing it